versions.txt

```diff
@@ -1,3 +1,4 @@
+3.0 - Single player with minimal setup is fully functional with SQLite database
```

Comment:
Casually releasing DLUv3 :)

Let's actually make sure to do a proper GitHub release once the "drop-and-play" changes are in.